source/code-snippets/usage-examples/findOne.js

```diff
@@ -0,0 +1,22 @@
+
+const { MongoClient } = require('mongodb');
+
+
+const client = new MongoClient(
+   'mongodb+srv://<user>:<password>@<cluster-url>?retryWrites=true&w=majority',
+);
+
+async function run() {
+   try {
+      await client.connect();
+      const database = client.db('sample_mflix');
+      const collection = database.collection('movies');
+      const query = { title: 'The Room' };
+      const movie = await collection.findOne(query);
+      // since this method returns the matched document, not a cursor, print it directly
+      console.log(movie);
+   } finally {
+      await client.close();
```

Comment:
IMO this client management code is relatively distracting and not very true to actual development. Devs rarely write just a single database function (unless maybe it's a Lambda?) and I probably want to pool connections across my entire app. AFAIK every call of `MongoClient.connect()` creates a connection pool, so opening it on a function call seems wasteful - I'd more likely have it at a higher-level scope or import from a module.

All that to say - the core of this example is `const movie = await collection.findOne(query);` and we should make sure to emphasize that.